tools/meson.build

```diff
@@ -0,0 +1,56 @@
+sources = files(
+  'klvanc_util.c',
+  'demo.c',
+  'parse.c',
+  'smpte2038.c',
+  'scte104.c',
+  'genscte104.c',
+  'gensmpte2038.c',
+  'eia708.c',
+  'smpte12_2.c',
+  'afd.c',
+  'udp.c',
+  'url.c',
+  'ts_packetizer.c',
+  'klringbuffer.c',
+  'pes_extractor.c',
+)
+
+thread_dep = dependency('threads')
+
+foreach exe_name: [
+  'klvanc_util',
+  'klvanc_parse',
+  'klvanc_smpte2038',
+  'klvanc_scte104',
+  'klvanc_genscte104',
+  'klvanc_gensmpte2038',
+  'klvanc_eia708',
+  'klvanc_smpte12_2',
+  'klvanc_afd',
+]
+  exe = executable(exe_name,
+    sources,
```

Comment:
This is a multicall binary, so you don't really *need* to install multiple of them. You could symlink or hardlink them with a `meson.add_install_script()` or, in meson 0.61.0, `install_symlink()`.

But you could also compile the sources into a utility static_library() and link them into individual executables. Meson will compile each source once per `executable()` since it may theoretically have different options, and because the outputs are contained inside private directories per-target rather than having e.g. dozens of files in builddir/tools/ and only one of them being an executable.

Using a utility static_library would mean that (not counting the linker stage for each tool) 15 build edges x 9 tools would reduce down to just 15 shared build edges.